### PR TITLE
Step-69: Use SolutionTransfer instead of low-level boost archive

### PR DESCRIPTION
**Pull request by tamiko**
Closes: #12558

### CODE REVIEW

File: examples/step-69/step-69.cc [hunk 1]
```diff
@@ -2464,11 +2469,28 @@ namespace Step69
...
+      if (resume)
+        {
+          discretization.refinement = 0;
+          discretization.setup();
+          discretization.triangulation.load(base_name + "-checkpoint.mesh");
+        }
+      else
+        {
+          discretization.setup();
+        }
```
Comment (marcfehling) [inline examples/step-69/step-69.cc:2493]:
```suggestion
      if (resume)
        {
          discretization.setup();
          discretization.triangulation.load(base_name + "-checkpoint.mesh");
        }
      else
        {
          discretization.setup();
          discretization.triangulation.refine_global(discretization.refinement);
        }
```

I would suggest to move the `refine_global()` part out of the `setup()` function. This way, it is more clear what `setup()` actually does in my opinion.

Technically both meshes are still refined `discretization.refinement` times. With the suggestion, there is no need to set it to 0.

**Review by marcfehling — APPROVED** [review]
Looks good to me! I just have one suggestion.

File: examples/step-69/step-69.cc (hunk 1 above)
Comment (tamiko) [reply examples/step-69/step-69.cc:2493]: Sounds good, will do.

**Comment by tamiko** [comment]
@marcfehling Suggestion commited, rebased and reindented.

**Review by marcfehling — APPROVED** [review]

**Pull request merged**